src/NLU.DevOps.Core.Tests/LabeledUtteranceConverterTests.cs

```diff
@@ -121,6 +121,29 @@ public static void DeserializesDerivedType()
             actual.Entities[0].Score.Should().Be(entityScore);
         }
 
+        [Test]
```

Comment:
You probably don't need this example if you just change L193 to "3" instead of 4 (this will also test endPos < startPos)